templates/search/blocks/search_sort_and_view_options.html

```diff
@@ -14,7 +14,7 @@ <h2 class="tna-heading-l search-sort-view__heading">
 
         {% render_fields_as_hidden form exclude='sort_by' %}
 
-        <input type="submit" value="Update" class="search-sort-view__form-submit">
+        <input type="submit" value="Sort by" class="search-sort-view__form-submit">
```

Comment:
```suggestion
        <input type="submit" value="Sort" class="search-sort-view__form-submit">
```
same as previous comment.

Reply:
Thanks @TNA-Allan , let's go with 'Sort by' for now and I'll raise with Alex in the future if thay wish to change it.